PolyEngine/Engine/Src/ShaderProgram.cpp

```diff
@@ -123,5 +123,5 @@ void ShaderProgram::SetUniform(const String& name, const Vector& val)
 
 void ShaderProgram::SetUniform(const String& name, const Matrix& val)
 {
-	glUniformMatrix4fv(m_uniforms[name], 1, GL_FALSE, val.GetTransposed().Data);
+	glUniformMatrix4fv(m_uniforms[name], 1, GL_FALSE, val.GetTransposed().Data.data());
```

Comment:
please create `float* GetDataPtr()` method in all math types changed by this PR to make this more convinient